docs/source/ngv.rst

```diff
@@ -0,0 +1,20 @@
+Neuro-Glia-Vasculature (NGV)
+============================
+
+.. image:: img/ngv.drawio.svg
+   :alt: NGV Diagram
+   :align: center
+
+The NGV simulation requires three node populations, each stored in separate files: **neocortex_neurons**, **astrocytes**, and **vasculature**.
+
+- **Neocortex_neurons** represent standard neurons.
```

Comment:
not all, but the neurons "located in the neocortex", right?

Reply:
uhm yes. It is a little redundant since we can load neurodamus-models with just one model (so all the neurons are neocortex no?) but I can add it

Reply:
The ngv mod files are inside the neurodamus-models/neocortex folder, there are other regions thalamus, hippocampus. Technically the ngv works only with neocortex neurons.